tests/system/action/projector_countdown/test_create.py

```diff
@@ -0,0 +1,59 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class ProjectorCountdown(BaseActionTestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        self.set_models(
+            {
+                "meeting/1": {"projector_countdown_default_time": 11},
+                "projector_countdown/1": {"title": "blablabla", "meeting_id": 1},
+            }
+        )
+
+    def test_create(self) -> None:
+        response = self.request(
+            "projector_countdown.create",
+            {
+                "meeting_id": 1,
+                "title": "test",
+                "description": "good description",
+                "default_time": 30,
+            },
+        )
+        self.assert_status_code(response, 200)
+        model = self.get_model("projector_countdown/2")
+        assert model.get("title") == "test"
+        assert model.get("meeting_id") == 1
+        assert model.get("description") == "good description"
+        assert model.get("default_time") == 30
+        assert model.get("countdown_time") == 30
+
+    def test_create_title_not_unique(self) -> None:
+        response = self.request(
+            "projector_countdown.create",
+            {
+                "meeting_id": 1,
+                "title": "blablabla",
+                "description": "good description",
+                "default_time": 30,
+            },
+        )
+        self.assert_status_code(response, 400)
+        assert "Title already exists in this meeting." in response.data.decode()
+
+    def test_create_no_default_time(self) -> None:
+        response = self.request(
+            "projector_countdown.create",
+            {
+                "meeting_id": 1,
+                "title": "test2",
+                "description": "good description",
+                "default_time": None,
+            },
+        )
+        self.assert_status_code(response, 200)
+        model = self.get_model("projector_countdown/2")
+        assert model.get("title") == "test2"
+        assert model.get("default_time") == 11
+        assert model.get("countdown_time") == 11
```

Comment:
Please add a test where no default time is given (not even `None`). This should also lead to a default_time of 11

Reply:
Updated the test: Remove the default_time: None and generate a new model without default of default_time, so the default can be generated from the meeting.